ironfish/package.json

```diff
@@ -57,8 +57,8 @@
     "lint": "tsc -b && tsc -b tsconfig.test.json && eslint --ext .ts,.tsx,.js,.jsx src/",
     "lint:fix": "tsc -b && tsc -b tsconfig.test.json && eslint --ext .ts,.tsx,.js,.jsx src/ --fix",
     "start": "tsc -b -w",
-    "test": "tsc -b && tsc -b tsconfig.test.json && cross-env NODE_OPTIONS=--experimental-vm-modules jest --testTimeout=${JEST_TIMEOUT:-5000}",
-    "test:slow": "tsc -b && tsc -b tsconfig.test.json && cross-env NODE_OPTIONS=--experimental-vm-modules TEST_INIT_RUST=true jest --testMatch \"**/*.test.slow.ts\" --testPathIgnorePatterns --testTimeout=${JEST_TIMEOUT:-60000}",
+    "test": "tsc -b && tsc -b tsconfig.test.json && cross-env NODE_OPTIONS=--experimental-vm-modules RUST_BACKTRACE=1 jest --testTimeout=${JEST_TIMEOUT:-5000}",
+    "test:slow": "tsc -b && tsc -b tsconfig.test.json && cross-env NODE_OPTIONS=--experimental-vm-modules TEST_INIT_RUST=true RUST_BACKTRACE=1 jest --testMatch \"**/*.test.slow.ts\" --testPathIgnorePatterns --testTimeout=${JEST_TIMEOUT:-60000}",
```

Comment:
Consider using:

```
RUST_BACKTRACE=${RUST_BACKTRACE:-1}
```

which means: set `RUST_BACKTRACE` to `1` only if it's not already set. This way, if someone wants to use e.g. `RUST_BACKTRACE=full`, it'll pass through.

Reply:
That's a good idea, but we need to switch to a different cross-env with support for that otherwise it won't work on windows. This cross-env supports that syntax. https://www.npmjs.com/package/@naholyr/cross-env

I can open another PR to do that.